clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -5403,6 +5403,65 @@ def AtomicCmpXchg : CIR_Op<"atomic.cmp_xchg",
   let hasVerifier = 0;
 }
 
+def MemScope_SingleThread : I32EnumAttrCase<"MemScope_SingleThread",
+                                            0, "single_thread">;
+def MemScope_System       : I32EnumAttrCase<"MemScope_System",
+                                            1, "system">;
+
+def MemScopeKind : I32EnumAttr<
+    "MemScopeKind",
+    "Memory Scope Enumeration",
+    [MemScope_SingleThread, MemScope_System]> {
+  let cppNamespace = "::cir";
+}
+
+def AtomicOrdering_Relaxed
+  : I32EnumAttrCase<"AtomicOrdering_Relaxed",
+                    0, "relaxed">;
```

Comment:
We already have the atomic order defined elsewhere, please re-use that!

Reply:
It is indeed defined, although `include "mlir/Dialect/LLVMIR/LLVMEnums.td"` throws an error of an already existing `AsmATT`:

```shell
error: def already exists: AsmATT
def AsmATT : I32EnumAttrCase<"x86_att", 0>;
    ^
note: location of previous definition
def AsmATT : LLVM_EnumAttrCase<
    ^
ninja: build stopped: subcommand failed.
```

Reply:
I mean the ones defined in CIR already, not the LLVM ones :)